tailwind.config.js

```
@@ -1,10 +1,16 @@
 module.exports = {
 	// NOTE: Update this to include the paths to all of your component files.
-	content: ["./app/**/*.{js,jsx,ts,tsx}", "./components/**/*.{js,jsx,ts,tsx}"],
+	content: [
+		"./app/**/*.{js,jsx,ts,tsx}",
+		"./components/**/*.{js,jsx,ts,tsx}",
+		"./constants/**/*.{js,jsx,ts,tsx}",
+	],
 	presets: [require("nativewind/preset")],
 	theme: {
 		colors: {
+			white: "#FFFFFF",
```

Biome (1.9.4)

[error] This property value named white is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named white


(lint/suspicious/noDuplicateObjectKeys)